extensions.json

```
@@ -634,6 +654,11 @@
       "repository": "https://github.com/swellaby/vscode-rust-test-adapter",
       "version": "0.11.0"
     },
+    {
+      "id": "tchayen.markdown-links",
+      "repository": "https://github.com/tchayen/markdown-links",
+      "version": "0.6.0"
```

Since there are no dedicated release tags or branches, it's ok to publish straight from the default branch without pinning a specific version:

```suggestion
      "repository": "https://github.com/tchayen/markdown-links"
```